comp/agent/autoexit/autoexitimpl/manager.go

```diff
@@ -71,6 +71,7 @@ func startAutoExit(ctx context.Context, sd exitDetector, log log.Component, tick
 	lastConditionNotMet := time.Now()
 	ticker := time.NewTicker(tickerPeriod)
```

Comment:
How about moving this line declaring the ticker down one line so that it's both within the go-routine and closer to the defer.

Reply:
Yep fair, I initially tried to minimize the changes, but that seems better 👍